app/pb_kits/playbook/pb_typeahead/_typeahead.scss

```diff
@@ -0,0 +1,52 @@
+.pb_typeahead {
+  .pb_typeahead_wrapper {
+    position: relative;
+  }
+
+  .text_input_wrapper {
+    margin-bottom: 0;
+  }
+
+  .pb_item_kit {
+    padding: 10px 0;
```

Comment:
We should probably use spacing tokens from `app/pb_kits/playbook/tokens/_spacing.scss` here instead of static px values.

Reply:
@thestephenmarshall What value would you suggest?

Reply:
Try `$space_xs + 2 0`